packages/runtime/test/cli/start.mjs

```diff
@@ -1,5 +1,8 @@
 import { parseArgs } from 'node:util'
 import { create } from '../../index.js'
+import { createRuntime } from '../helpers.js'
+
+const createFn = process.env.PLT_USE_PLAIN_CREATE ? create : createRuntime
```

Comment:
Why?

Reply:
The tests in this folder were previously using `plt-runtime` which no longer exists. I replaced with a fake CLI to retain them.